src/ec/packages/ec-component-table/table.html.twig

```diff
@@ -0,0 +1,102 @@
+{% spaceless %}
+{#
+  Parameters:
+  - "zebra" (boolean) (default: false))
+  - "headers" (array) (default: []): format: [
+      {
+        "label" (string or array of string)
+        "colspan" (string) (default: ''),
+        "data-ecl-table-header-group" (string) (default: ''),
+      },
+      ...
+    ]
+  - "rows" (array) (default: []): format: [
+      {
+        "label" (string or array of string)
+        "data-ecl-table-header" (string) (default: ''),
+        "group" (bool) (default: false),
+      },
+      ...
+    ]
+  - "extra_classes" (string) (default: '')
+  - "extra_attributes" (array) (default: []): format: [
+      {
+        "name" (string) (default: ''),
+        "value" (string) (default: '')
+      },
+      ...
+    ]
+#}
+
+{# Internal properties #}
+
+{% set _css_class = 'ecl-table' %}
+{% set _zebra_css_class = 'ecl-table--zebra' %}
+{% set _head_css_class = 'ecl-table__head' %}
+{% set _header_css_class = 'ecl-table__header' %}
+{% set _body_css_class = 'ecl-table__body' %}
+{% set _cell_css_class = 'ecl-table__cell' %}
+{% set _cell_header_group_class = 'ecl-table__cell--group' %}
+{% set _row_css_class = 'ecl-table__row' %}
+{% set _headers = headers|default({}) %}
+{% set _rows = rows|default([]) %}
+{% set _zebra = zebra|default(false) %}
+
+{#  Internal logic - Process properties#}
+
+{% if extra_classes is defined and extra_classes is not empty %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_classes %}
+{% endif %}
+
+{% if zebra %}
```

Comment:
Better if you use the internal variable:
```suggestion
{% if _zebra %}
```

Otherwise, just remove `_zebra`'s declaration

Reply:
@yhuard Good catch.